python/pyspark/broadcast.py

```diff
@@ -246,9 +349,20 @@ def clear(self) -> None:
         self._registry.clear()
 
 
-if __name__ == "__main__":
+def _test() -> None:
```

Comment:
```suggestion
def _test():
```

This is fine. PySpark only types the public API only :-).

Reply:
without ` -> None`, python lint test failed

Reply:
interesting ..